infrastructure/kube/keep-dev/tbtc-dapp-service.yaml

```diff
@@ -3,15 +3,17 @@ apiVersion: v1
 kind: Service
 metadata:
   name: tbtc-dapp
-  namespace: tbtc
+  namespace: default
+  annotations:
   labels:
     app: tbtc
     type: dapp
 spec:
+  type: NodePort
```

Comment:
For my own edification, what's the type if you don't specify it, as before? Or is it this, but we're wanting to be explicit?

Reply:
`ClusterIp`, I've gone back and forth on being explicit even if we're using the default type.